### DISCUSSION TITLE
How to disable default Submit button?

### DISCUSSION THREAD

**Discussion by FactorD** [team-forms/community#24] [Q&A]
Hi all,

We have a multi-page form and in most scenarios we utilize the default Submit button which shows on the last visible page. But, there are some scenarios in which this default Submit button is not wanted.

Is there any way to deactivate this functionality for certain pages? Or a condition we can set under which this button doesn't show?

Thank you and best regards,
Damon

**Comment by AnthonyPhan** [comment]
Unfortunately, there's no specific setting for conditionally hiding the submit button. However, you can still hide it using a custom CSS selector as shown below.

![image](https://github.com/team-forms/community/assets/12785492/84de7008-43fc-4174-a5d7-daa78b09d515)


```CSS
.btn-wizard-nav-submit{
    display:none;
}
```
